misc/CCSynth/CC/DataInsights/src/prose_datainsights.egg-info/requires.txt

```diff
@@ -0,0 +1,6 @@
+dotnetcore2>=2.1.8
```

Comment:
not sure about having dotnetcore in requires.txt here. Again, I think the .egg-info directory can go, if we are releasing the sources and not the compiled binaries.